tests/fixtures/phpcs/DrupalClassCorrect.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\testmodule;
+
+use Drupal\BlockBase;
+
+/**
+ * Some service.
+ */
+class DrupalClassCorrect extends BlockBase {
+
+  /**
+   * Using \Drupal here but it should be injected instead.
+   */
+  public function test(): bool {
+    return TRUE;
+  }
+
+  /**
+   * Tests method with docblock and PHP attribute.
+   */
+  public function aMethodWithDocblockAndPhpAttribute(): void {
```

Comment:
Methods are descriptive, nice, but we need to have the correct naming

Reply:
Fixed